docs/en/latest/FAQ.md

```diff
@@ -76,3 +76,13 @@ kubectl get crd apisixroutes.apisix.apache.org -o jsonpath='{ .spec.versions[*].
 If you can not find `v2beta3` in `ApisixRoute` definition file. Please apply the latest version of `ApisixRoute`.
 
 Ref to FAQ #7.
+
+### 9. Modify the Admin API key in APISIX-Ingress
+
+Usually, you need to modify the Admin API key to protect Apache APISIX and APISIX Ingress. Please refer to this [link](https://apisix.apache.org/zh/docs/apisix/how-to-build/) to simply change Apache APISIX.
+
+However, in apisix-ingress-controller, if we need to change the Admin API key, we also need to change the Admin API key in apisix-ingress-controller. There are two different ways to implement the requirements here.
+
+For the first method, we need to modify the Admin API credentials values in both the `apisix/values.yaml` and `apisix/apisix-ingress-controller/values.yaml` files. You can refer to these two links(apisix's [values.yaml](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/values.yaml#:~:text=admin%3A%20edd1c9f034335f136f87ad84b625c8f1) && apisix-ingress-controller's [values.yaml](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix-ingress-controller/values.yaml#:~:text=adminKey%3A%20%22edd1c9f034335f136f87ad84b625c8f1%22)).
+
+Another method, you can just pass `--set ingress-controller.config.apisix.adminKey=<Your new admin key>`  to `helm install` command.
```

Comment:
In the first method, you modify the admin key of APISIX and APISIX Ingress, but in the second method, you only modify the admin key of APISIX Ingress.
Please keep them consistent.

Reply:
Solved!